README.md

```diff
@@ -1 +1,71 @@
-# Redux tools
\ No newline at end of file
+# Redux tools
+
+A collection of Redux libraries for modular applications, such as those utilizing [React-union](https://github.com/lundegaard/react-union) or other SPA-in-CMS solutions. They allow you to easily:
+
+- tie reducers and epics to your widgets and handle their lifecycle, i.e. injection and ejection, automatically.
+- isolate your widgets' Redux state while still allowing them to communicate with one another.
+
+They offer seamless integration with React and React-union. See the [packages](/packages) and the [FAQ](FAQ.md) for more details!
+
```

Comment:
Can I ask you about brief section `Packages`? It should contain just list of packages with short description. Thanks!

Reply:
Of course, I had that in mind, but I totally forgot. Thanks!